assets/scripts/src/choices.js

```diff
@@ -1261,7 +1261,7 @@ class Choices {
     });
 
     if (choice && !choice.selected && !choice.disabled) {
-      const canAddItem = this._canAddItem(activeItems, choice.value);
+      const canAddItem = this._canAddItem(activeItems, choice.label);
```

Comment:
This doesn't make sense to me. `_canAddItem` expects the value of the choice to determine whether it can be added as an item 🤔

Reply:
At all cases, which I tried all is working fine for me and it resolves #377 when value might be not string value. Label is always a string and in test cases, which I added to this fix you can see, that I'm searching choice by the label and it's added successfully, but if I leave `choice.value` it'll throw an error.
Could you provide me with case, where my changes break something?

Reply:
So if you're label and value are different values, this will cause an issue. This is more covering up a smell elsewhere - the actual problem is that the code is assuming the value is a string

Reply:
Please re-consider your position on this. I explained in further detail @ https://github.com/jshjohnson/Choices/pull/378#issuecomment-392547687